test/ViewModel/Converter/ModelConverterTestCase.php

```diff
@@ -113,29 +113,27 @@ private function findSamples() : Traversable
                     $list = true;
                     $version = 3;
                     break;
-                case 'podcast-episode-chapter':
-                    $model = 'recommendations';
-                    $type = true;
-                    $list = true;
-                    break;
                 case 'external-article':
+                case 'podcast-episode-chapter':
                     $model = 'recommendations';
                     $type = true;
                     $list = true;
                     break;
                 case 'annual-report':
-                case 'article-poa':
-                case 'article-vor':
                 case 'blog-article':
                 case 'collection':
                 case 'event':
                 case 'interview':
                 case 'labs-post':
                     $version = 2;
                     break;
+                case 'article-poa':
                 case 'press-package':
                     $version = 3;
                     break;
+                case 'article-vor':
+                    $version = 6;
+                    break;
             }
 
             $samples = Finder::create()->files()->in(ComposerLocator::getPath('elife/api')."/dist/samples/{$model}/v{$version}");
```

Comment:
I think an improvement we should make is if the we cannot find a folder that we expect to be there then we should throw an exception. This way we should not find ourselves in this position again!